.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_PRODUCT_API = http://127.0.0.1:8000/products/api/
```

Comment:
No need to move all API endpoints to the `.env` file, endpoints don't change with different environments only the domain does. You can move all endpoints to constants which will prepend domain from `.env`. We also don't commit the `.env` file every environment has its own local `.env` so put this file in `.gitignore` and commit another file `.env.example` with all required environment variables for reference.